outputs.tf

```diff
@@ -1,14 +1,8 @@
-# Just for testing
 output "openai_endpoint" {
   description = "The endpoint used to connect to the Cognitive Service Account."
   value       = azurerm_cognitive_account.this.endpoint
 }
 
-output "openai_key" {
```

Comment:
This `openai_key` should be used when calling Azure OpenAI from outside of Azure. It gives the same openai.com experience. There is also a secondary key.
Either we expose both as sensitive values or we expose none of them in the outputs.

Reply:
I would recommend expose this key as a sensitive output since we don't limit how our users call the API. They can call it by using identity, or the key.

Reply:
we agreed that we will expose both of them as sensitive values.